### ISSUE TITLE
getTransaction & getTransactions diverged

### ISSUE THREAD

**Issue by hairtail** [iron-fish/ironfish#1976]
### What happened?

Get completed in `accounts:transactions` while get pending in `accounts:transactions --hash=xxx`. 
As described in https://github.com/iron-fish/ironfish/pull/1813
<img width="1436" alt="截屏2022-08-08 上午8 48 29" src="https://user-images.githubusercontent.com/82858858/183318612-3517d748-12f0-45f2-8a4a-ab33f47dcb6e.png">

<img width="956" alt="image" src="https://user-images.githubusercontent.com/82858858/183318494-e60104f2-e775-4cce-8a5b-3ae42ddf1941.png">

### Version

0.1.43

### Debug output

```shell
Cli ux display bug, No more debug information.
```

### Relevant log output

```shell
Cli ux display bug, No more debug information.
```

### Graffiti

miningKing

**Comment by wd021** [comment]
good find. about to start work on https://github.com/iron-fish/ironfish/issues/1805. will address this with it

**Comment by hughy** [comment]
Fixed in #1883
